docs/crocks/Pair.md

```diff
@@ -0,0 +1,85 @@
+# Pair
+
+`Pair c a`
+
+--
+
+--
+
+```js
+--
+```
+
+`Pair` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`of`](#of), [`type`](#type) | [`ap`](#ap), [`bimap`](#bimap), [`chain`](#chain), [`concat`](#concat), [`equals`](#equals), [`fst`](#fst), [`inspect`](#inspect),  [`map`](#map), [`merge`](#merge), [`of`](#of), [`type`](#type), [`snd`](#snd), [`swap`](#swap), [`value`](#value) |
+
+## Constructor
+
+### of
+
+`Pair m => c a -> m c a`
+
+### type
+
+`() -> String`
+
+## Instance
+
+### ap
+
+`Pair m => m c (a -> b) ~> m c a -> m c b`
+
+### bimap
+
+`Pair m => m c a ~> (c -> d) -> (a -> b) -> m d b`
+
+### chain
+
+`Pair m => m c a ~> (a -> m c b) -> m c b`
+
+### concat
+
+`Pair m => m c a ~> m c a -> m c a`
```

Comment:
Need to show `Semigroup` constraint on  both. this is tricky as both could vary in their `Semigroup`
(check source code)